docs/README.md

```diff
@@ -104,7 +104,7 @@ deck.gl is developed in parallel with a number of companion modules, including:
 
 * [luma.gl](https://luma.gl/) - A general purpose WebGL library designed to be interoperable both with the raw WebGL API and (as far as possible) with other WebGL libraries. In particular, luma.gl does not claim ownership of the WebGL context, and can work with any supplied context, including contexts created by the application or other WebGL libraries.
 * [loaders.gl](https://loaders.gl) - a suite of framework-independent loaders for file formats focused on visualization of big data, including point clouds, 3D geometries, images, geospatial formats as well as tabular data.
-* [react-map-gl](https://visgl.github.io/react-map-gl/) - A React wrapper around Mapbox GL which works seamlessly with deck.gl.
+* [react-map-gl](https://visgl.github.io/react-map-gl/) - A React wrapper around Mapbox GL which works seamlessly with deck.gl. There are two integration modes to choose from depending on which features you need: either as a container holding deckgl layers (allowing more react-map-gl features such as controls and terrain), or as deck.gl's child interpreting deck.gl's view state (allowing more Deck features, such as multi-view and effects).
```

Comment:
What would be helpful link here for documentation or an example about these two modes?

Reply:
The mention of different interaction model feels like to much detail here as this is list is intended for the user to make a quick scan., I would just provide a link to the details.

In the same veing, I would also drop the verbiage about luma.gl not owning the WebGL context (in the bullet just above), as it seems to specific for this list.

 In fact we may want to do a doc wide search for luma.gl  / WebGL references and tone done down overly WebGL specific wordings

Reply:
I've made an edit related for react-map-gl.